v1/api/swagger/components/schemas/Host.yaml

```
@@ -191,3 +192,8 @@ allOf:
         description: Host health summary status
       Labels:
         $ref: ./Labels.yaml
+      WWPNs:
+        type: array
+        description: FC host port world wide port numbers
```

WWPN should be "world wide port names" -
```suggestion
        description: FC host port world wide port names
```